velox/exec/HashProbe.h

```diff
@@ -410,6 +455,12 @@ class HashProbe : public Operator {
 
     // True if currentRow has a match.
     bool currentRowPassed{false};
+
+    // If the currentRow at the end of an output batch has not passed the
```

Comment:
```
// If set, it points to the last missed (input) row carried over from previous output batch processing. The carried row is either written as a passed row if the same input row has a hit in the next output batch processing or written to the first next output batch which has space at the end if it can never have a hit. 
```